db.go

```diff
@@ -2068,13 +2087,24 @@ func (d *DB) EstimateDiskUsage(start, end []byte) (uint64, error) {
 					)
 				}
 				if err != nil {
-					return 0, err
+					return 0, 0, 0, 0, err
+				}
+				totalFiles++
+				if file.Virtual {
+					remoteFiles++
```

Comment:
Virtual =/= remote unfortunately, so this will have to be `meta.IsShared()` below (which would, under new terminology, more accurately be called `IsRemote()`)

Ps. you can always look up a file in the provider, it doesn't need to be a virtual file to do this indirection.

Reply:
done